packages/crdt/src/crdt/merkle.ts

```diff
@@ -43,7 +49,7 @@ export function insert(trie: TrieNode, timestamp: Timestamp) {
   let hash = timestamp.hash();
   let key = Number(Math.floor(timestamp.millis() / 1000 / 60)).toString(3);
 
-  trie = Object.assign({}, trie, { hash: trie.hash ^ hash });
+  trie = Object.assign({}, trie, { hash: (trie.hash || 0) ^ hash });
```

Comment:
Before: `hash` was allowed to be `null`.

After: in strict mode `hash` can no longer be null. So we must cast it to a number here. I went with 0 because that seems to be the default value

```
console.log(undefined ^ 5); // 5
console.log(null ^ 5); // 5
console.log(0 ^ 5); // 5
```